kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -57,7 +61,10 @@
         return nameSpace;
       },
       rooturl() {
-        return this.defaultFile.storage_url;
+        const isOS = iOSTest.test(navigator.userAgent);
+        const iOSorIE11 = isOS || IE11Test.test(navigator.userAgent);
```

Comment:
```suggestion
        const iOS = iOSTest.test(navigator.userAgent);
        const iOSorIE11 = iOS || IE11Test.test(navigator.userAgent);
```

Reply:
Haha, that was fun, "this constant indicates if the user has an operating system" :)